### PR TITLE
my_work

### PR DESCRIPTION
**Pull request by Slava-11**

### CODE REVIEW

File: web/2022/Gondor/src/scss/index.scss [hunk 1]
```diff
@@ -0,0 +1,175 @@
+@import './reset.scss';
+
+
+body{
```
Comment (mikerudenko) [inline web/2022/Gondor/src/scss/index.scss:4]: розбити на декілька файлів - header.scss, main.scss, footer.scss

File: web/2022/Gondor/src/scss/index.scss [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+.header{
```
Comment (mikerudenko) [inline web/2022/Gondor/src/scss/index.scss:7]: code is not formatted, please format code before push

File: web/2022/Gondor/src/scss/index.scss [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    display: flex;
+    justify-content: space-between;
+     
+    .circle_img1{
+        height: 100px;
+        width: 100px;
+        border-radius: 50%;
+    }
+    .header_flex_ul{
+        li{
```
Comment (mikerudenko) [inline web/2022/Gondor/src/scss/index.scss:24]: you should use only class selectors, selectors by tag are bad practise

File: web/2022/Gondor/src/scss/index.scss [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+        background-color:  aqua;
+        position: relative;
+        animation-name: circle;
+        animation-duration: 7s;
+        animation-timing-function: linear;
+        animation-iteration-count:  infinite;
+        animation-direction: alternate;
+         
+    }
+    @keyframes circle{
```
Comment (mikerudenko) [inline web/2022/Gondor/src/scss/index.scss:133]: please create separate file animation.scss

File: web/2022/Gondor/src/index.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+<head>
+<title>Gondor</title>
+<link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+    <div class="header">
+        <div class="header_flex">
+            <div class="header_flex_circle"><img class="circle_img1" src="img/img1.jpg.crdownload" alt=""/></div>
+            <div class="header_flex_ul">
+                <ul>
```
Comment (mikerudenko) [inline web/2022/Gondor/src/index.html:11]: please add class to tags - ul and li

File: web/2022/Gondor/src/index.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
+<html>
+<head>
+<title>Gondor</title>
+<link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+    <div class="header">
```
Comment (mikerudenko) [inline web/2022/Gondor/src/index.html:7]: please format html

File: web/2022/Gondor/src/index.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        <div class="header_flex">
```
Comment (mikerudenko) [inline web/2022/Gondor/src/index.html:8]: classes should naming with understandable meaning

Comment (mikerudenko) [reply web/2022/Gondor/src/index.html:8]: you should NOT use css in class names

File: web/2022/Gondor/src/index.html [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+            <div class="header_flex_circle"><img class="circle_img1" src="img/img1.jpg.crdownload" alt=""/></div>
```
Comment (mikerudenko) [inline web/2022/Gondor/src/index.html:9]: please rename all your images

File: web/2022/Gondor/src/index.html [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+            <div class="nav1_flex_element_img2"><img src="img/img2.jpg" alt=""></div>
+            <div class="nav1_flex_element_text">
+                <ul>
+                    <li class="li_title">Історія</li>
+                    <li class="li_text">Наприкінці Другої Епохи, після падіння Нуменора і порятунку Вірних Нуменорців на чолі з Еленділом, кораблі синів Еленділа Ісілдура і Анаріона занесло на південь біля гирла Великої Річки Андуін. Піднявшись угору течією річки, вони заснували володіння, що згодом отримало назву Гондор (Кам'яна Земля). Столицею Королівства стало місто Осгіліат на обох берегах Андуіну, з'єднане великим мостом з вежами та кам'яними будинками. Обабіч столиці стояли дві твердині: Мінас-Ітіль (Вежа Східного Місяця) на схилі Гір Тіні як погроза Мордору та Мінас-Анор (Вежа Спадного Сонця) біля підніжжя Бескиду Міндоллуін як захист від дикунів долин</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div class="nav2">
```
Comment (mikerudenko) [inline web/2022/Gondor/src/index.html:33]: nav1, nav2, nav3 are bad namings - please use meaning classes like

Google name - minas-tirith-section, aragorm-section, footer-section

**Review by mikerudenko — CHANGES_REQUESTED** [review]

**Comment by Slava-11** [comment]
understood accepted processed

**Review by mikerudenko — APPROVED** [review]

**Pull request merged**